src/components/error-boundary/ErrorBoundary.tsx

```diff
@@ -1,35 +1,23 @@
-import { captureException } from '@sentry/react-native';
-import React, { PropsWithChildren } from 'react';
-// @ts-ignore
-import { IS_TESTING } from 'react-native-dotenv';
+import React from 'react';
+import * as Sentry from '@sentry/react-native';
 import Fallback from './Fallback';
-import logger from '@/utils/logger';
+import { IS_TEST } from '@/env';
+import { useTheme } from '@/theme';
 
-class ErrorBoundary extends React.Component<PropsWithChildren> {
-  static getDerivedStateFromError(_error: any) {
-    return { hasError: true };
-  }
+const NoErrorBoundary = ({ children }: { children: React.ReactNode }) => children;
 
-  state = { hasError: false };
+const ErrorBoundaryWithSentry = ({ children }: { children: React.ReactNode }) => {
+  const { colors } = useTheme();
+  return (
+    <Sentry.ErrorBoundary
+      beforeCapture={scope => scope.setTag('RainbowErrorBoundary', 'true')}
```

Comment:
@brunobar79 we can track these tags on Sentry. Or we can look up the message `RainbowAppRestartFromErrorBoundary` within RainbowError